stdlib/.depend

```diff
@@ -103,6 +103,11 @@ stdlib__callback.cmx : \
     stdlib__obj.cmx \
     stdlib__callback.cmi
 stdlib__callback.cmi :
+camlinternalAtomic.cmo : \
+    camlinternalAtomic.cmi
+camlinternalAtomic.cmx : \
+    camlinternalAtomic.cmi
+camlinternalAtomic.cmi :
```

Comment:
@gadmm those are the lines that CheckDepend complains about, I believe. I will try to improve the check to show the difference.

Reply:
Well-spotted. So you have proof that I _did_ try with CamlinternalAtomic ;) Their source were committed at some point so I must have done some bad manoeuvre with git. Showing the diff in the test would definitely have helped, thanks.